src/pages/departures.tsx

```diff
@@ -16,25 +17,52 @@ function DeparturesContent({ autocompleteFeatures }: DeparturesContentProps) {
   const { t } = useTranslation();
   const [query, setQuery] = useState('');
   const result = useAutocomplete(query);
+  const [selectedPosition, setSelectedPosition] = useState<
+    Position | undefined
+  >();
```

Comment:
This might be temporary as we don't have selected stop place, but for future reference I think this should come from some selected venue/stop place/address/position with something like:

```ts
const selectedPlace = useSelectedPlaceByUrl();

//...

<Map position={selectedPlace.position} />
```

Where `useSelectedPlaceByUrl` gets ID from URL and fetches specific place by id.

Reply:
Yes, I was thinking that this was just temporary to have something to test the map with. Thinking about it, it might not be the best to have such code merged into the main branch 🤔 I'm trying to divide the Mapbox issue into multiple PRs and make this only the initial integration of Mapbox, but might want to increase the scope. Not sure

Reply:
I think this is task is more relevant for the issue with searching for actual places.